generator/src/lib.rs

```diff
@@ -2877,7 +2877,25 @@ pub fn write_source_code<P: AsRef<Path>>(vk_headers_dir: &Path, src_dir: P) {
         bindings = bindings.allowlist_type(typ);
     }
 
+    #[derive(Debug)]
+    struct ParseCallbacks;
+    impl bindgen::callbacks::ParseCallbacks for ParseCallbacks {
+        fn enum_variant_behavior(
+            &self,
+            _enum_name: Option<&str>,
+            original_variant_name: &str,
+            _variant_value: bindgen::callbacks::EnumVariantValue,
+        ) -> Option<bindgen::callbacks::EnumVariantCustomBehavior> {
+            if original_variant_name.ends_with("_MAX_ENUM") {
+                Some(bindgen::callbacks::EnumVariantCustomBehavior::Hide)
+            } else {
+                None
+            }
```

Comment:
I just realized, this could be turned into `.then(|| bindgen::callbacks::EnumVariantCustomBehavior::Hide)` :hand_over_mouth: 

Been told before that that's a bit too tricky for the average Rust dev though, leaving it out.

Reply:
I'm partial to cheeky combinators personally, but it's a matter of taste :grinning:

Reply:
Many loathe control flow hidden behind functions, functional style :D